addon/mixins/frost-list-mixin.js

```diff
@@ -44,12 +44,12 @@ export default Mixin.create(FrostListSelectionMixin, FrostListExpansionMixin, Fr
     )
   }),
 
-  listMixinConfig: Ember.computed('activeSorting', 'sortableProperties', 'sortedItems.[]', function () {
+  listMixinConfig: Ember.computed('activeSorting', 'sortableProperties', 'statefulListItems.[]', function () {
     let activeSorting = this.get('activeSorting')
```

Comment:
These `let`s aren't necessary, the get can be put directly into the return object